### PR TITLE
Add feedback from Chris. Many thanks!

### PR DESCRIPTION
**Pull request by tomschr**
@ChrisKowalczyk
These are the changes from your feedback. I think I added them all. Could you have a final look?
I think, I've confused myself with the "crm status" output, so please take a closer look.

Thanks a lot for your help! 👍

### CODE REVIEW

File: xml/art_sle_ha_pmremote.xml [hunk 1]
```diff
@@ -360,7 +360,7 @@
   <sect2 xml:id="sec.ha.pmremote.config.uc1.remote.nodes">
    <title>Configuring Remote Nodes</title>
    <para>
-    The following procedure creates a guest node <systemitem
+    The following procedure prepares the remote node <systemitem
       class="domainname">&node4;</systemitem> on your cluster node
     <systemitem class="domainname">&node3;</systemitem>:
```
Comment (gao-yan) [inline xml/art_sle_ha_pmremote.xml]: The following procedure prepares a remote node &node3;

Comment (tomschr) [inline xml/art_sle_ha_pmremote.xml]: Ok, fixed. 👍

Comment (gao-yan) [inline xml/art_sle_ha_pmremote.xml]: There's no node "doro" in use case 1.

Comment (tomschr) [inline xml/art_sle_ha_pmremote.xml]: Right, I've removed it, so the sentence is: "The following procedure prepares your cluster node
     <systemitem class="domainname">&node3;</systemitem>:"

File: xml/art_sle_ha_pmremote.xml [hunk 2]
```diff
@@ -454,13 +454,21 @@ ssh: connect to host &node3; port 3121: Connection refused</screen>
...
-      should contain a running cluster with nodes that are all accessible.
+      should contain a running cluster with nodes that are all accessible:
      </para>
+     <screen>&prompt.root;<command>crm</command> status
+[...]
+Online: [ alice bob ]
+RemoteOnline: [ charlie ]
+
+Full list of resources:
+  [...]</screen>
```
Comment (tomschr) [inline xml/art_sle_ha_pmremote.xml]: Right. Added. :)

File: xml/art_sle_ha_pmremote.xml [hunk 3]
```diff
@@ -707,10 +715,10 @@ GuestOnline: [ &node4;@&node1; ]
 
 Full list of resources:
 vm-doro (ocf:heartbeat:VirtualDomain): Started &node1;
-fake1           (ocf:pacemaker:Dummy): Started &node4;
-fake2           (ocf:pacemaker:Dummy): Started &node2;
+fake1           (ocf:pacemaker:Dummy): Started &node1;
+fake2           (ocf:pacemaker:Dummy): Started &node1;
 fake3           (ocf:pacemaker:Dummy): Started &node1;
-fake4           (ocf:pacemaker:Dummy): Started &node2;</screen>
+fake4           (ocf:pacemaker:Dummy): Started &node1;</screen>
```
Comment (tomschr) [inline xml/art_sle_ha_pmremote.xml]: Ok, great. Changed it in commit aa4bc56.

**Pull request closed without merging**